core/src/main/java/io/questdb/griffin/SqlParser.java

```diff
@@ -2609,25 +2725,26 @@ private void parseUpdateClause(
     }
 
     @NotNull
-    private ExecutionModel parseWith(GenericLexer lexer, SqlParserCallback sqlParserCallback) throws SqlException {
-        parseWithClauses(lexer, topLevelWithModel, sqlParserCallback);
+    private ExecutionModel parseWith(GenericLexer lexer, SqlParserCallback sqlParserCallback,
```

Comment:
nit: we need to avoid this type of code formatting, with multiple args on the same line while also using multi-line formatting